var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -199,9 +200,16 @@ def install(self, spec, prefix):
             # FIXME: On some systems, the installed binaries for METIS cannot
             # be executed without first being read.
             ls = which('ls')
-            ls('-all', prefix.bin)
+            ls('-a', '-l', prefix.bin)
```

Comment:
Good call; I'll amend my previous change so that this is only executed if tests are run.